### DISCUSSION TITLE
Is it possible to re-order channel or sort them alphabetically?

### DISCUSSION THREAD

**Discussion by flo-assling** [globaleaks/globaleaks-whistleblowing-software#4559] [Q&A] [answered]
### Proposal

At the moment it seems that channels cannot be sorted after creating.

### Motivation and context

This means that channels need to be created in the order they shall appear.
If we an another channel later, it will always be the last.
It would be a great advantage to be able to sort them afterwards.

**Comment by evilaliv3** [comment, marked as answer]
Thank you @flo-assling 

By default channels are sorted authomatically but you can disable this option inside the advanced settings. Look for the option "Display contexts in alphabetical order".

p.s. Context was the previous term used to indicate channels. I'm going to correct this old refusal now.
